pyproject.toml

```diff
@@ -53,6 +53,10 @@ classifiers = [
 test = [
     "pytest",
     "pytest-asyncio",
+    "cloudpickle",
+    "dask",
+    "distributed",
+    "cupy-cuda11x",
```

Comment:
This is presumably making explicit what was previously implicit

Reply:
Yes exactly, and it's important for the user experience if they do `pip install ucx-py` to get all the dependencies pulled in (and we leverage it in the tests of the wheels now as well).